flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlNodeToOperationConversion.java

```diff
@@ -365,6 +381,11 @@ private static Optional<Operation> convertValidatedSqlNode(
             return Optional.of(converter.convertDelete((SqlDelete) validated));
         } else if (validated instanceof SqlUpdate) {
             return Optional.of(converter.convertUpdate((SqlUpdate) validated));
+        } else if (validated instanceof SqlAlterModel) {
+            return Optional.of(
+                    converter.alterModelConverter.convertAlterModel((SqlAlterModel) validated));
+        } else if (validated instanceof SqlDropModel) {
+            return Optional.of(converter.convertDropModel((SqlDropModel) validated));
```

Comment:
Is there anything that blocks from moving it to SqlNodeConverters instead of putting here?